charts/bundler/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: bundler
 description: A Open Policy Agent (OPA) Data Bundle Server providing permissionable data from ISPyB
 type: application
 version: 0.3.0
-appVersion: 0.0.12
+appVersion: 0.0.11
```

Comment:
Shouldn't version numbers go up not down?

Reply:
Yes, but the repo is only on tag `0.0.11`, not sure how it got to `0.0.12` :sweat_smile:

Reply:
Okay, I [do know](https://github.com/DiamondLightSource/authz/commit/89ad2324d316898b51c98ae8363488faec2e4279), but I don't know why you bumped it